atomicfu-transformer/src/main/kotlin/kotlinx/atomicfu/transformer/AtomicFUTransformerJS.kt

```diff
@@ -64,11 +65,32 @@ class AtomicFUTransformerJS(
 
     // erase getting value of atomic field
     private fun AstNode.eraseGetValue(): String {
-        val res = this.toSource()
+        var res = this.toSource()
```

Comment:
You have `AstNode` here. Why do you really need to convert it to string a parse it again via regeexps?

Reply:
In the AST I use for transformation, it's possible to transform nodes only without changing their type. "Get value" nodes like these: 
```
arr.get$atomicfu(3).kotlinx$atomicfu$value
x.kotlinx$atomicfu$value
```
are of type PropertyGet: `x` is a target field, `kotlinx$atomicfu$value` is a property field and after transformation, it should be of type Name: just `x`. So, I extract all property get calls via regeexps now.